cypress/run-e2e.sh

```diff
@@ -11,45 +11,20 @@ finally() {
 
   local exit_code="${1:-0}"
   # This is the clean up.
-  # Find any node processes running from within the client dir    
+  # Find any client or server node processes  
```

Comment:
Would this actually be  "# Find any api, client, or server node processes" ? I was actually wondering about this before. I pushed a commit on this hopefully to clarify this. Thank you!!

Reply:
Yep, that's better, thanks for fixing it.